### PR TITLE
Using the light sensor

### PR DESCRIPTION
**Pull request by amitkle**

### CODE REVIEW

File: app/src/main/java/com/example/coronadiagnosticapp/ui/activities/OxymeterActivity.java [hunk 1]
```diff
@@ -45,6 +39,7 @@
     // Variables Initialization
     private static final String TAG = "HeartRateMonitor";
     private static final AtomicBoolean processing = new AtomicBoolean(false);
+    private static final int MIN_LIGHT_VALUE = 170; // lux units.
```
Comment (svechinsky) [inline app/src/main/java/com/example/coronadiagnosticapp/ui/activities/OxymeterActivity.java]: How did you choose this value?

File: app/src/main/java/com/example/coronadiagnosticapp/ui/activities/OxymeterActivity.java [hunk 2]
```diff
@@ -472,12 +505,12 @@ private void removeProgressBar() {
     }
 
     private void removeProgressBarAndShowAlert() {
-        alert.setVisibility(View.VISIBLE); // Make alert "no finger" - visible
+        put_finger_alert.setVisibility(View.VISIBLE); // Make alert "no finger" - visible
```
Comment (svechinsky) [inline app/src/main/java/com/example/coronadiagnosticapp/ui/activities/OxymeterActivity.java]: I think we want to add a check if the measurement started already (did the user put their finger at took it off after) or it is waiting to start (the user hasn't put their finger on yet)
We should raise different alerts in each case

File: app/src/main/java/com/example/coronadiagnosticapp/ui/activities/OxymeterActivity.java (hunk 1 above)
Comment (amitkle) [reply app/src/main/java/com/example/coronadiagnosticapp/ui/activities/OxymeterActivity.java]: just some basic tests I made. The researchers will need to choose a better value

Comment (svechinsky) [reply app/src/main/java/com/example/coronadiagnosticapp/ui/activities/OxymeterActivity.java]: ok

**Comment by svechinsky** [comment]
Note, I fixed a small import conflict

**Pull request merged**